src/components/dialogmote/referat/Stillingsprosent.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+import { useAppSelector } from "../../../hooks/hooks";
+import { latestSykmeldingForVirksomhet } from "../../../utils/sykmeldinger/sykmeldingUtils";
+
+const texts = {
+  stillingsprosent: "Stillingsprosent:",
+};
+
+const EkstraInfo = styled.div`
+  margin-bottom: 1em;
+`;
+
+const BoldText = styled.span`
+  font-weight: bold;
+  margin-right: 1em;
+`;
+
+interface StillingsprosentProps {
+  virksomhetsnummer: string;
+}
+
+const Stillingsprosent = ({ virksomhetsnummer }: StillingsprosentProps) => {
+  const sykmeldingerState = useAppSelector((state) => state.sykmeldinger);
+  const sykmeldingerData = sykmeldingerState.data;
+  const newestSykmeldingForVirksomhet = latestSykmeldingForVirksomhet(
```

Comment:
Kan eventuelt lage en custom hook, så slipper man utils metode. useNewestSykmeldingForVirksomhet()

Reply:
Tja, jeg kan eventuelt lage en hook som gir sykmeldingdata, så slipper jeg å bruke to linjer på det. Men det blir jo samme kode for å finne riktig sykmelding, så jeg vet ikke om det er mer naturlig å ha det i en hook, enn i en util 🤷‍♂️

Reply:
Mest hipt med hooks 🧔